util/httpUtil.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	userAgent = "metrics-collector/1.0"
+	userAgent = "dcos-metrics-collector/1.0"
```

Comment:
TODO (at a future date): set the version (e.g., `1.0`) in `dcos-metrics-collector.go` and use that constant here, instead of burying the version somewhere in the `util` package.